### DISCUSSION TITLE
How to represent backslashes in Micron?

### DISCUSSION THREAD

**Discussion by parnikkapore** [markqvist/Reticulum#120] [Nomad Network]
How do I write a Micron document such that a backslash appears in the output?

Here are a few things I've tried and how they render in the app.

```text
Source:   A \` B \\` C \ D \\ E ` F
Rendered: A ` B ` C  D  E ` F
```

**Comment by Swissbandit** [comment]
Ill look into the guide as soon im home,
The literal is '=
 then after that you can enter whatever
And it will display it as typed.
'= until ended

**Reply by Swissbandit** [reply]
In the guide of nomadnet,
Under the section markup,
Mark has many examples and the later half is the guide toppic as literal block,to see how it was written.

**Reply by parnikkapore** [reply]
Unfortunately, it looks like `= does not work if used for part of a line.

**Comment by parnikkapore** [comment]
More concretely, how do I write the following line in a Micron document?

> **Side note:** You might need to double escape \\ characters in your string.

**Reply by parnikkapore** [reply]
Here are a few attempts:

![Some unsuccessful attempts at writing a backslash within a formatted line](https://user-images.githubusercontent.com/9170353/195984701-e73928a3-71a7-47f0-a698-ec64b006cdc2.png)

**Reply by Swissbandit** [reply]
Maybe it will work when there is a space between the command '= and the / and after  '=  as if not it wont execute the command

**Reply by markqvist** [reply]
Haha, oh man, there is something funky going on here. I'll need to have a look at the parser and see what is up.

**Reply by Swissbandit** [reply]
this is also the reason for breaking any anscii art using it.
and i suspect there are some more symbols with same issue.

**Reply by markqvist** [reply]
Yep, we need to map this out and find a good overall solution!

**Comment by markqvist** [comment]
It is fixed now :) Micron uses `\` to escape backticks, but I had forgotten to let the parser also escape `\` itself.

So you can now render a `\` by writing `\\` in the micron source.
